tests/Gedmo/Mapping/MappingEventAdapterTest.php

```diff
@@ -49,19 +44,14 @@ public function testCorrectAdapter(): void
 
     public function testAdapterBehavior(): void
     {
-        $eventArgsMock = $this->getMockBuilder(LifecycleEventArgs::class)
-            ->disableOriginalConstructor()
-            ->getMock();
-        $eventArgsMock->expects(static::once())
-            ->method('getObjectManager');
+        $emMock = $this->createStub(EntityManagerInterface::class);
+        $entity = new \stdClass();
 
-        $eventArgsMock->expects(static::once())
-            ->method('getObject')
-            ->willReturn(new \stdClass());
+        $args = new PrePersistEventArgs($entity, $emMock);
 
         $eventAdapter = new EventAdapterORM();
-        $eventAdapter->setEventArgs($eventArgsMock);
-        $eventAdapter->getObjectManager();
-        $eventAdapter->getObject();
+        $eventAdapter->setEventArgs($args);
+        static::assertSame($eventAdapter->getObjectManager(), $emMock);
+        static::assertInstanceOf(\stdClass::class, $eventAdapter->getObject());
```

Comment:
Since we are testing behavior, instead of expecting that the `EventArgs::getObjectManager()` and `EventArgs::getObject()` are called, we can check with the result of these calls.